tests/tom-swifty-test/SwiftReflector/BindingImportTests.cs

```diff
@@ -23,7 +23,7 @@ public void LoadsDatabaseSmokeTest (PlatformName platform, int expectedLowerLimi
 			var database = importer.Import ();
 			Assert.IsNotNull (database, $"null database for {platform}");
 			Assert.Less (expectedLowerLimit, database.Count, $"Expected at least {expectedLowerLimit} db entries, but got {database.Count} entries.");
-			Assert.IsTrue (!errors.AnyErrors, $"{errors.ErrorCount} errors importing database.");
+			errors.AssertNoErrors ("errors importing database.");
```

Comment:
an error with `$"{errors.ErrorCount} errors while {whileDoing}"` and `"errors importing database."` will give you

`"1 errors while errors importing database."`